compute/cloud-client/src/test/java/compute/disks/DisksFromSourceIT.java

```diff
@@ -59,8 +58,8 @@
 public class DisksFromSourceIT {
 
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static String ZONE;
-  private static String REGION;
+  private static final String ZONE = "asia-south1-a";
+  private static final String REGION = ZONE.substring(0, ZONE.length() - 2);
```

Comment:
Same question here. Why are we hard-coding?
Applies to multiples files in this PR.

Reply:
Answered in previous comment.